openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -43,7 +43,8 @@ def _capture(self, preset):
                     json.dumps(preset, indent=4, sort_keys=True)
                 )
             )
-
+        if "textures" in preset["viewport_options"]:
+            lib.reload_textures()
```

Comment:
Ouch. I do not want this to happen by default actually for all my playblasts - even if I _were_ to playblast with textures. We have UDIM tile regenerations done on scene open and before the exports.

Do we have other options?

Maybe this force refresh should be a setting of its own?

Reply:
I think it could be the setting of its own but it might be happened only when the texture placement is enabled.

Reply:
Sure - but we playblast with udims quite often and haven't had issues - the UV tile regeneration has always been quite slow in maya so if we can avoid it - I'd rather be able to.